Neos.Media.Browser/Configuration/Settings.yaml

```diff
@@ -27,6 +27,10 @@ Neos:
         # By default, enable the option to create redirects for replaced asset resources
         createAssetRedirectsOption:
           enable: true
+        # By default, disable showing the workspace owner/title in media browser usage tab
+        showWorkspaceOwnerOrName:
+          enable: false
+          privilegedRoles: ['Neos.Neos:Administrator']
```

Comment:
Any reason why you didn't introduce an actual privilege instead of defining a privilege via the settings?

You could introduce a new privilege and integrators can add it to whichever role they want.
Could also be enabled by default for admins IMO.

Reply:
Good point, haven't thought about that one. 

I've implemented the privilege but im stuck trying to check the role without looping over them.
Would be nice to have something like `Account::hasRole()` including parent roles.

Or should i just loop over the user (parent) roles and match them with the new privilege role.

Reply:
You can either use `$this->securityContext->hasRole($roleIdentifier)` or `$this->privilegeManager->isPrivilegeTargetGranted($privilegeTarget);`

Reply:
Thanks worked like a charm!